packages/api/src/api/expoJudgingSession/get.ts

```diff
@@ -9,7 +9,7 @@ export const get = async (req: Request, res: Response) => {
     const expoJudgingSessions = await entityManager.find(
       ExpoJudgingSession,
       {},
-      { orderBy: { createdAt: QueryOrder.DESC } },
+      { orderBy: { createdAt: QueryOrder.ASC } },
```

Comment:
i dont know how to frame this question properly, but i am trying to understand why did you make this ascending?

Reply:
ascending will order the sessions by oldest to newest. when done here it will be performed on the DB server preventing the client or app server from doing the extra work.